src/main/java/Parser/WeekFormatParse.java

```diff
@@ -0,0 +1,23 @@
+package Parser;
+
+import Commons.LookupTable;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class WeekFormatParse {
+    private static LookupTable LT = new LookupTable();
```

Comment:
name wise i think can change lookupTable. LT seems like its violating the naming standard.

Reply:
Changing to lookupTable might confuse with the class LookupTable, maybe can try use a more distinct name